appengine/hello-world/standard/package.json

```diff
@@ -15,15 +15,17 @@
   "scripts": {
     "deploy": "gcloud app deploy",
     "start": "node app.js",
-    "system-test": "repo-tools test app",
+    "system-test": "mocha --exit test/*.test.js",
     "test": "npm run system-test",
     "e2e-test": "repo-tools test deploy"
   },
   "dependencies": {
     "express": "^4.16.3"
   },
   "devDependencies": {
-    "@google-cloud/nodejs-repo-tools": "^3.3.0"
+    "@google-cloud/nodejs-repo-tools": "^3.3.0",
```

Comment:
I think we can delete this dependency now?

Reply:
No :) It's used in the e2e-test. But turns out, we're not calling 2e2-test from CI, instead we have special test files that run a deployment. But I'll prefer deleting it in a separate PR. There's a lot of magic happening in `repo-tools` based on env variables.